topic_monitor/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+setup(
+    name='topic_monitor',
+    version='0.0.0',
+    packages=[],
+    py_modules=[
+        'launch_files.launch_depth_demo',
+        'launch_files.launch_durability_demo',
+        'launch_files.launch_fragmentation_demo',
+        'launch_files.launch_reliability_demo',
+        'scripts.data_publisher',
+        'scripts.topic_monitor',
```

Comment:
that's a good point. I didn't notice because I always symlink install. they get installed under the package's namespace now: https://github.com/ros2/demos/pull/90/commits/1147a7622b3449172bdc3c561049d1a27cf6012b